certbot/tests/lock_test.py

```diff
@@ -3,6 +3,10 @@
 import multiprocessing
 import os
 import unittest
+try:
+    import fcntl
+except ImportError:
+    fcntl = None  # type: ignore
```

Comment:
This is probably overkill, but can we do something similar to what we do in `lock.py` of setting a `bool` here? I just like avoiding these `type: ignore` lines because unlike disabling linter checks, disabling type checking on a variable will disable additional checks around where the variable is used and I'd like us to get the most out of mypy.

Reply:
Done, also.